index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.4/css/bulma.min.css" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Project-1</title>
+</head>
+
+<body>
+    <section class="columns">
+        <div class="column is-3">
+            <div class="column is-12 has-background-warning mt-6 ml-6">
+                <div>
+                    <label class="column is-12 p-1">Find concerts for any artist or city:</label>
+                    <div>
+                        <input class="column is-12 p-1 mb-1" type="text" placeholder="Search Here">
+                        <button type="submit" class="btn column is-12 p-1">Search</button>
+                    </div>
+                </div>
+                <div class="has-background-warning container">
+                    <div class="has-background-white mt-1">
+                        search
```

Comment:
line 25, search meaning what? did you want that to be the button or the div where everything shows up?

Reply:
sorry I added a ul class to this div to replace "search", this is where I wanted the search results to appear